### DISCUSSION TITLE
【求助】关于西北边境地区进入绳网的问题

### DISCUSSION THREAD

**Discussion by sunnyt0wn** [share121/inter-knot#882] [其他]
我这里的所有未经授权的反屏蔽通道都失效了，不知道是光缆遇上了空洞还是治安局开始严打了，更换通道、设备、不同公司的接入点都不行。不同于沿海地区，绳网在我这里被完全屏蔽了，现在我只能通过官方授权的Watt Tookit来连接绳网，但是速度极其感人，最高速度只有3Mbps每秒。有相同经历的人吗？

**Comment by zhangyj0523** [comment]
有钱自己搭个梯子，从来没卡过

**Comment by sunnyt0wn** [comment]
我感觉是协议被屏蔽了，这种情况下自己搭也不行啊，我用的ss和vmess都不行
________________________________
发件人: ***@***.*** ***@***.***> 代表 SugarLabyrinth ***@***.***>
发送时间: 2024年8月10日 19:27
收件人: share121/inter-knot ***@***.***>
抄送: sunnyt0wn ***@***.***>; Author ***@***.***>
主题: Re: [share121/inter-knot] 【求助】关于西北边境地区进入绳网的问题 (Discussion #882)


有钱自己搭个梯子，从来没卡过

―
